### PR TITLE
Ibraheem-Peer session overlay form

### PR DESCRIPTION
**Pull request by Ibhassan01**
Screenshot (52)

### CODE REVIEW

**Review by 7malikk — DISMISSED** [review]

**Comment by 7malikk** [comment]
LGTM

File: development/src/components/PeerOverlay_comp.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+
+function PeerSession() {
+    const [activeLanguage, setActiveLanguage] = useState('');
+    return (
+        <div className="bg-[#2F2F3A] text-white p-6">
+            <div className=" flex flex-col">
+                <div className="flex justify-between">
+                    <div className=" font-bold text-3xl leading-8 font-nohemi">New Peer Session</div>
+                    <button><Image src={closeIconWhite} className="w-5 h-5" /></button>
```
Comment (annonymousauthority) [inline development/src/components/PeerOverlay_comp.js:18]: Put some close function here, so we know. Don't keep the button bare

File: development/src/components/PeerOverlay_comp.js
Comment (annonymousauthority) [inline development/src/components/PeerOverlay_comp.js:1]: Implement dark theme and light theme for this component

File: development/src/components/PeerOverlay_comp.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                            onClick={() => setActiveLanguage('css')}
+                            className={` w-32 h-32 flex justify-center items-center shadow-md shadow-black flex-col rounded-md ${activeLanguage === 'css'
+                                    ? 'bg-blue-500 text-white '
+                                    : ' bg-gray-200 dark:bg-[#3D3D48]'
+                                }`}>
+                            <Image src={CSS} alt="language-icon" className="w-8 h-8 mb-4" />
+                            <p className="text-white font-nohemi font-bold text-xl">CSS</p>
+                        </div>
+                        <div
+                            onClick={() => setActiveLanguage('js')}
```
Comment (annonymousauthority) [inline development/src/components/PeerOverlay_comp.js:53]: active language should be `javascript` not `js`

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

File: development/src/components/PeerOverlay_comp.js (hunk 2 above)
Comment (Ibhassan01) [reply development/src/components/PeerOverlay_comp.js:53]: Alright

**Comment by Ibhassan01** [comment]
Screenshot (63)

**Review by annonymousauthority — APPROVED** [review]
LG

**Review by annonymousauthority — APPROVED** [review]
LG

**Review by annonymousauthority — APPROVED** [review]
LG

**Pull request merged**